src/components/PdfOverview/PdfOverview.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { Document, Page } from 'react-pdf'
+
+interface PdfOverviewProps {
+  file: File
+}
+
+const styles = {
+  container: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center'
+  }
+}
+
+export const PdfOverview: React.FC<PdfOverviewProps> = ({
+  file
+}: PdfOverviewProps) => {
+  const parentRef = useRef<HTMLDivElement | null>(null)
+  const [parentWidth, setParentWidth] = useState(0)
+
+  useEffect(() => {
+    const updateSize = (): void => {
+      if (parentRef.current) {
+        const width = parentRef.current.getBoundingClientRect().width
+        setParentWidth(width)
+      }
+    }
+    updateSize()
+  }, [])
```

Comment:
How does it behave when you resize the window?

Reply:
Not necessary here, I think.
Do you have a use case in mind?

Reply:
Not really, it was just to be sure it was not forgot